Kitodo-Docket/src/main/java/org/kitodo/docket/Docket.java

```diff
@@ -29,9 +29,14 @@ public File generateDocket(DocketData docketData, URI xslFileUri) throws IOExcep
         File file = new File("docket.pdf");
         FileOutputStream fileOutputStream = new FileOutputStream(file);
 
-        exportDocket.startExport(docketData, fileOutputStream, new File(xslFileUri));
-        fileOutputStream.flush();
-        fileOutputStream.close();
+        try {
```

Comment:
Please use [try-with-resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) statement instead of handling this manually.